lib/ansible/modules/cloud/digital_ocean/do_sshkeys.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
```

Comment:
@ryansb IMHO all that names are to be too big but I understand the problem of keep consistency. Can I suggest an alias to **digital_ocean_sshkeys**? (or rename to *digital_ocean_sshkeys* and create and alias to *do_sshkeys*)